cypress/integration/code-schools.spec.js

```diff
@@ -14,4 +14,31 @@ describe('code schools', function() {
     // 40 is arbitrary, but it proves that the API is working and leading to rendered content
     cy.get('[data-testid="SchoolCard"]').should('have.length.greaterThan', 40);
   });
+
+  it('should render Cincy and Tech Elevator after selecting Ohio', () => {
+    cy.wait('@codeSchools');
```

Comment:
Seeing it occur in writing makes me realize we could just put `cy.wait('@codeSchools')` at the end of the `beforeEach()`.

Reply:
Thats what I was thinking.

Reply:
Completed this